src/framegrab/cli/autodiscover.py

```diff
@@ -23,7 +23,7 @@
 @click.option(
     "--rtsp_discover_modes",
     type=click.Choice(PREVIEW_RTSP_COMMAND_CHOICES, case_sensitive=False),
-    default="light",
+    default="disable",
```

Comment:
Might be better to set it to disable since we don't want to guess passwords by default.

Reply:
I think we _do_ want to guess passwords in `autodiscover`. Without the password, we cannot connect to the grabber, which is the point of the `autodiscover`, right? I guess there is the question of how thorough we are with our password guessing; maybe that should be an option?

Reply:
Yea, the mode `light` should be alright but other modes such as `complete_fast` and `complete_slow` may cause you to exceed the number of attempts before the camera blocks you.